lightning/backend/tidb.go

```diff
@@ -228,17 +239,37 @@ func (enc *tidbEncoder) appendSQL(sb *strings.Builder, datum *types.Datum, col *
 
 func (*tidbEncoder) Close() {}
 
+func getColumnByIndex(cols []*table.Column, index int) *table.Column {
+	if index == len(cols) {
+		return extraHandleTableColumn
+	} else {
+		return cols[index]
+	}
```

Comment:
```suggestion
	return cols[index]
```

strange that it is not linted